l10n_it_ricevute_bancarie/views/account_view.xml

```diff
@@ -115,13 +115,27 @@
         <field name="model">account.invoice</field>
         <field name="inherit_id" ref="account.invoice_form"/>
         <field name="arch" type="xml">
+            <xpath expr="//div[@groups='base.group_multi_currency']" position="after">
```

Comment:
Visto che parliamo di pagamenti, non avrebbe più senso visualizzare questo campo vicino al termine di pagamento?

Reply:
Grazie del suggerimento @SimoRubi

Reply:
ok quindi ora il nuovo campo viene dopo `cash_rounding_id` che, come dice l'`help`:
> Defines the smallest coinage of the currency that can be used to pay by cash.

In che modo questo campo è collegabile alle RiBa insolute?
Non avrebbe più senso visualizzare questo campo vicino al termine di pagamento, visto che esistono termini di pagamento che citano espressamente le RiBa (ad esempio nei dati demo di questo modulo)?

Reply:
Le stesse osservazioni valgono anche per la vista `tree` poco sotto, solo che lì non ci sono campi direttamente correlati al metodo di pagamento, forse il meno peggio sarebbe vicino a `date_due` ma anche dove l'hai messo credo sia accettabile

Reply:
Per la `tree` mi è stato espressamente chiesto di metterlo lì (prima delle cifre).
Volevo precisare che nell'intestazione della fattura non c'è nessun campo che rimanda alle Ri.ba. insolute, solo il termine di pagamento. Il campo 'Insoluta' è invisibile se è `False` e ho pensato che messo in fondo avrebbe dato maggiormente nell'occhio.
Credo però sia più corretto come dici tu.